spec/features/user_sees_all_horses_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../spec_helper.rb'
+
+describe "when a user visits the horse index" do
+  it "they see a list" do
+    Horse.create(name: "jon", age: 30, total_winnings: 44)
+    Horse.create(name: "sally", age: 8, total_winnings: 900)
+    visit "/horses"
+
+    expect(page).to have_content("jon")
+    expect(page).to have_content("sally")
```

Comment:
These expectations are good, but they can be more robust. To begin, try scoping where you're looking for the content by css selectors. ex:

```ruby
within(".horse_info") do
  expect(page).to have_content("jon")
  expect(page).to have_content("sally")
end
```

When you scope where in the HTML the expectation is finding the content you're writing a more robust test that makes assertions about the structure of the html as well as what is actually printed to the page.